packages/metamask/package.json

```diff
@@ -60,7 +60,7 @@
     "typescript": "^5.2.2"
   },
   "dependencies": {
-    "@metamask/sdk": "^0.20.5",
+    "@metamask/sdk": "^0.26.0",
```

Comment:
The package version on line 3 needs an alpha version bump and also needs to be reflected and tested in `packages/demo/package.json`

Reply:
<img width="941" alt="Screenshot 2024-06-21 at 10 25 25" src="https://github.com/blocknative/web3-onboard/assets/61094771/d5be30ea-b8d8-4c2b-9856-8a95cf6baa80">

Reply:
After updating the MetaMask SDK package to version "2.1.3-alpha.1", I encounter an issue when I go to the demo test dapp and try to run yarn after also updating MetaMask there to "@web3-onboard/metamask": "^2.1.3-alpha.1". 

What is the problem?